pricing-page-exercise/merve/index.html

```diff
@@ -1,12 +1,126 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML 5 Boilerplate</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML 5 Boilerplate</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" />
+</head>
+
+<body>
+  <header>
+    <div class="left">
+      <span>Company name</span>
+    </div>
+    <div class="right">
+      <ul>
+        <li><a href="#">Features</a></li>
+        <li><a href="#">Enterprise</a></li>
+        <li><a href="#">Support</a></li>
+        <li><a href="#">Pricing</a></li>
+        <li><a href="#" class="signup">Sign up</a></li>
+      </ul>
+    </div>
+  </header>
+  <div class="pricing1">
+    <h1 class="pricing">Pricing</h1>
+  </div>
+  <div>
+    <p>Quickly build an effective pricing table for your potential customers with this Bootstrap example. It's built
+      with default Bootstrap components and utilities with little customization.</p>
+  </div>
+
+  <div class="container">
+    <div class="box">
+      <div>
+        <h2>Free</h2>
+      </div>
+      <div class="line"></div>
+      <div class="span1"><span class="dolar">$0</span><span class="mo">/ mo</span></div>
+      <div class="olli">
+      <ol>
+        <li>10 users included</li>
+        <li>2 GB of storage</li>
+        <li>Email support</li>
+        <li>Help center access</li>
+      </ol>
+    </div>
+      <button class="button2">Sign up for free</button>
```

Comment:
```suggestion
      <button type="button" class="button2">Sign up for free</button>
```

However it's better to use `<a>` element here as you use it outside a `form` element